packages/components/progress-stepper/src/Step/Step.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { cx } from 'emotion';
+import { type CommonProps } from '@contentful/f36-core';
+import { getStyles } from './Step.styles';
+import { DoneIcon, CloseIcon, WarningIcon } from '@contentful/f36-icons';
+
+export interface StepProps extends CommonProps {
+  /**
+   * @default incomplete
+   */
+  state?:
+    | 'active'
+    | 'error'
+    | 'warning'
+    | 'disabled'
+    | 'complete'
+    | 'incomplete';
```

Comment:
noting that we decided to change this prop name from 'variant' to 'state' after aligning with the technical writing team on prop naming. This prop indicates the state of the component as opposed to a variation of the component. Miro board review can be found [here](https://miro.com/app/board/uXjVNTG41Fo=/)